src/client/Microsoft.Identity.Client/Internal/Requests/SilentRequestHelper.cs

```diff
@@ -53,19 +53,20 @@ internal static void ProcessFetchInBackgroundAsync(Func<Task<AuthenticationResul
                 }
                 catch (MsalServiceException ex)
                 {
-                    string logMsg = $"Background fetch failed. Is AAD down? { ex.IsAadUnavailable()}. MsalServiceException {ex.ToString()}";
+                    string logMsg = $"Background fetch failed MsalServiceException. Is AAD down? { ex.IsAadUnavailable()}";
                     if (ex.StatusCode == 400)
                     {
-                        logger.Error(logMsg);
+                        logger.ErrorPiiWithPrefix(ex, logMsg);
                     }
                     else
                     {
-                        logger.Warning(logMsg);
+                        logger.WarningPiiWithPrefix(ex, logMsg);
                     }
                 }
                 catch (Exception ex)
                 {
-                    logger.Warning($"Background fetch failed Exception {ex.ToString()}");
+                    string logMsg = $"Background fetch failed Exception.";

```

Comment:
the 'Exception' at the end of the sentence does not feel right. Either drop it (as it would appear through the logging method) or `failed with exception`.